nbs/tutorials/methods.ipynb

```diff
@@ -11,10 +11,43 @@
    "cell_type": "markdown",
```

Comment:
<p>This will not work, and I don't want to encourage this usage for now. If the user wants to change config, it is probably  better to construct a new <code>VanillaCF</code> object, rather than updating the config in-place. </p><p> </p><p>Now, this is more like a hack, not a functionality that we are supporting. So probably, let's remove this part for now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
removed it for now